runtime/doc/lua.txt

```diff
@@ -447,29 +447,29 @@ Note that underscore-prefixed functions (e.g. "_os_proc_children") are
 internal/private and must not be used by plugins.
 
 ------------------------------------------------------------------------------
-VIM.LOOP                                                   *lua-loop* *vim.loop*
+VIM.UV                                                   *lua-loop* *vim.uv*
 
-`vim.loop` exposes all features of the Nvim event-loop. This is a low-level
+`vim.uv` exposes all features of the Nvim event-loop. This is a low-level
 API that provides functionality for networking, filesystem, and process
 management. Try this command to see available functions: >vim
 
-    :lua print(vim.inspect(vim.loop))
+    :lua print(vim.inspect(vim.uv))
 <
-Internally, `vim.loop` wraps the "luv" Lua bindings for the LibUV library;
+Internally, `vim.uv` wraps the "luv" Lua bindings for the LibUV library;
 see |luv-intro| for a full reference manual.
```

Comment:
With the name change, I think this paragraph should be rewritten along the lines of 

"`vim.uv` exposes the "luv" Lua bindings for the libUV library that Nvim uses for networking, filesystem, and process management, see |luvref.txt|. In particular, it allows interacting with the main Nvim |luv-event-loop|."

(No need for the vimscript snippet, now that we have a full reference manual.)